README.md

```diff
@@ -117,104 +203,50 @@ See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art o
-
-Once the build has completed you can launch several instances of the built executable in order to both host and join a game.
-
-> Mac users: to run multiple instances of the same app, you need to use the command line.
-> Run `open -n BossRoom.app`
-
----------------
-**Multiplayer over internet**
-
-To play over internet, we need to build an executable that is shared between all players. See the previous section.
-
-It is possible to connect between multiple instances of the same executable OR between executables and the editor that produced said executable.
-
-Running the game over internet currently requires setting up a relay. Boss Room provides an integration with [Unity Relay](https://docs-multiplayer.unity3d.com/netcode/current/relay/relay).
-
-> Checkout our Unity Relay setup guide [here](https://docs-multiplayer.unity3d.com/netcode/current/relay/relay)
-
-Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. Make sure your host's address listens on 0.0.0.0 (127.0.0.1 is for local development only).
-
-------------------------------------------
-
-## Exploring the project
-BossRoom is an eight-player co-op RPG game experience, where players collaborate to take down some imps, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by mouse button or hotkey.
-
-One of the eight clients acts as the host/server. That client will use a compositional approach so that its entities have both server and client components.
-
-The game is server-authoritative, with latency-masking animations. Position updates are done through NetworkedVars that sync position, rotation and movement speed. NetworkedVars and Remote Procedure Calls (RPC) endpoints are isolated in a class that is shared between the server and client specialized logic components.
-
-Code is organized in domain based assemblies. See our [ARCHITECTURE.md](ARCHITECTURE.md) file for more details.
-
----------------
 
 For a deep dive in Unity Netcode and Boss Room, visit our [docs site](https://docs-multiplayer.unity3d.com/).
 
 ## Other samples
 ### Bite-size Samples
-This repository contains a collection of bitesize sample projects and games that showcase different sub-features of NGO. You can review these samples with documentation to understand APIs and features better.
-- [Our various bitesize samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)
+The [Bitesize Samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)  repository is currently being expanded and contains a collection of smaller samples and games, showcasing sub-features of NGO. You can review these samples with documentation to understand our APIs and features better.
```

Comment:
```suggestion
The [Bitesize Samples](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize)  repository contains a collection of smaller samples and games, showcasing sub-features of NGO. You can review these samples with documentation to understand our APIs and features better.
```
I wouldn't keep transitory statements in a readme like this, it's the best way to forget about those when update the bitesize and not boss room.

Reply:
I disagree with this one. I think we'll be working on BSS for at least the next year and this is good reminder for folks to keep checking back.